integration/commands_administration_test.go

```diff
@@ -1237,10 +1237,12 @@ func TestCollStatsCommandCount(tt *testing.T) {
 	AssertEqualDocuments(t, expectedComparable, actualComparable)
 }
 
-func TestCollStatsCommandScaleSize(t *testing.T) {
-	t.Parallel()
+func TestCollStatsCommandScaleSize(tt *testing.T) {
+	t := setup.FailsForFerretDB(tt, "https://github.com/FerretDB/FerretDB/issues/4792")
```

Comment:
I added a test script https://github.com/FerretDB/FerretDB/pull/4791/files to show case the difference.

Reply:
Yeah, that's what I should have done yesterday 😬